configs/stage/schemas/src/subscriptions.ksl

```diff
@@ -0,0 +1,14 @@
+version 0.1
+namespace subscriptions
+
+import rbac
+
+@rbac.add_v1_based_permission(app:'subscriptions', resource:'report', verb:'read', v2_perm:'subscriptions_report_view');
```

Comment:
@lindseyburnett Does this need to be "reports" instead of "report"?  Based on https://github.com/RedHatInsights/rbac-config/blob/master/configs/stage/permissions/subscriptions.json#L2